tests/test_covariances.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+import numpy as np
+import skfda
+
+
+class TestsSklearn(unittest.TestCase):
```

Comment:
El nombre del test no debería ser TestSklearnCovariances por ser del módulo que prueba

Reply:
The 'covariance' name already appears in the module name, so it seemed redundant to me.